tests/system/CMakeLists.txt

```diff
@@ -323,9 +327,21 @@ ENDIF()
                     0 # VMC
                     TRUE)
 
+  QMC_RUN_AND_CHECK(short-bccH_1x1x1_ae-vmc-all_sdj
```

Comment:
Could you add a comment here (and to the other added test) about the focus of the tests (all-electron moves).   The test names are necessarily short, but cryptic.   Plus there's a double use of 'all electron' - 'ae' is the all-electron potential (vs a pseudopotential) and 'all' for all-electron moves (vs. particle-by-particle moves)